src/app/containers/ATIAnalytics/params/cpsAssetPage/buildParams.js

```diff
@@ -4,6 +4,7 @@ import {
   getPublishedDatetime,
   LIBRARY_VERSION,
 } from '../../../../lib/analyticsUtils';
+import { getThingAttributes } from '../../../../lib/analyticsUtils/article';
```

Comment:
Could we use #lib/analyticsUtil instead of the ../ path notation?

Reply:
Also, should this live within analyticsUtils/article, or somewhere more generic?

Reply:
I personally think things like `#lib/analyticsUtils` just hides how fragmented the code in Simorgh is